src/embedding/kd.ts

```diff
@@ -186,7 +186,20 @@ export namespace EmbeddedKpiDashboard {
         /**
          * Type represent that the remove filter context action is finished
          */
-        RemoveFilterContextFinished = 'removeFilterContextFinished'
+        RemoveFilterContextFinished = 'removeFilterContextFinished',
+
+        /**
+         * Type that represents the drill to url was started and loading can be displayed. It does not contain url.
+         * Url is contained by DrillToUrlResolved which is posted after url is resolved. Event also contains ID which
```

Comment:
I see some grammar errors. Maybe rephrase it like this (or in similar manner, maybe my text is not without errors as well).

"Type that represents started drill to URL. The event does not contain an URL. The event can be used as notification to display a loading indicator as the URL resolving takes some time. 
The URL is sent in DrillToUrlResolved event which is posted after the URL is resolved. The event also contains an ID that can be matched with ID in subsequently posted DrillToUrlResolved event."